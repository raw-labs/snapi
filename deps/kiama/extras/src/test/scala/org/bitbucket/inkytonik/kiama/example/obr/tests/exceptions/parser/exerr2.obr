(* Missing identifier in CATCH *)

PROGRAM ExErr2 (i : INTEGER) : INTEGER;

BEGIN

   TRY
      Average := Quantity / Count;         
      Result := Average * Frequency;
   CATCH DO
      Result := 0;
   END
   
END ExErr2.
