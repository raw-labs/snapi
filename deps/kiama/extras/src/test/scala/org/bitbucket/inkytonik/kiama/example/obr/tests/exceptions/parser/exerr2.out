src/test/scala/org/bitbucket/inkytonik/kiama/example/obr/tests/exceptions/parser/exerr2.obr:11:7:error: keyword "DO" found where variable name expected
      Result := 0;
      ^
